### DISCUSSION TITLE
Improve stack trace in allure report

### DISCUSSION THREAD

**Discussion by MissiaL** [codeceptjs/CodeceptJS#3020] [General]
#### What are you trying to achieve?
I want to understand in which line of code the test fell in the allure report
#### What do you get instead?
I don’t get the full stack trace in the allure report and I don’t understand on which line of code the test fell

* CodeceptJS version: 2.4.3

My example of a fallen test
![image](https://user-images.githubusercontent.com/6458407/74713846-6f78f400-523a-11ea-9d24-5f99e7771ef7.png)

Why is there no link in the stack trace to the line where the test fell? I need to figure out where the test fell. This is inconvenient and time consuming.

**Comment by kobenguyent** [comment]
Shall be covered by allure lib.
